fly.toml

```diff
@@ -0,0 +1,30 @@
+app = "test-config-21"
```

Comment:
I don't think you meant to add this file

Reply:
Good catch